packages/apps/tools/.eslintrc.js

```diff
@@ -5,4 +5,9 @@ module.exports = {
   extends: ['@kadena-dev/eslint-config/profile/next'],
   parserOptions: { tsconfigRootDir: __dirname },
   ignorePatterns: ['**/generated/**'],
+  settings: {
+    'import/resolver': {
+      typescript: {}
+    },
+  },
```

Comment:
Should this `settings` be added to the `@kadena-dev/eslint-config/profile/next`?